### PR TITLE
The Jumpsuit Re-Detailening

### PR DESCRIPTION
**Pull request by Flareguy**
## About the PR
Makes the level-of-detail on color jumpsuits more similar to that of the ones seen in vgstation.
In layman's terms, the color jumpsuits are now more detailed.

## Why / Balance
A common complaint with the security resprite was that the jumpsuits were just too simplified - and to a degree, I think I have to agree. The current color jumpsuits feel really oversimplified and omitted a lot of detail that was there for a reason in the original.

The level-of-detail enhancement on these is not *immediately* apparent, but is still definitely noticeable by the naked eye, and will likely have a positive subconcious impact.

Also, because I know somebody is going to mention it: the weird diagonal lines on the jumpsuit don't exactly make sense as to what they're supposed to be (to me at least,) but definitely make the jumpsuit look better then without them, in my opinion. I intentionally left them on the engineering jumpsuit resprite I did almost 2 years ago because of this.

## Technical details
N/A

## Media
Content Client_vYt6bdQUx7

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
N/A

**Changelog**
no cl no fun

### CODE REVIEW

**Comment by beck-thompson** [comment]
Could you show a before and after? They look good but I don't know what I'm comparing it to 😆

File: Resources/Textures/Clothing/Uniforms/Jumpskirt/color.rsi/meta.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039 and modified by Flareguy for Space Station 14",
+  "copyright": "Taken from vgstation at commit https://github.com/vgstation-coders/vgstation13/commit/993de554ecfc30cc45f8340819c5a60ef1eee7e1 and modified for SS14 by Flareguy & P3MOIRA. Jumpskirt variants made using sprites tgstation at commit https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039 and modified by Flareguy for SS14",
```
Comment (UbaserB) [inline Resources/Textures/Clothing/Uniforms/Jumpskirt/color.rsi/meta.json:4]: Is it correct that the jumpskirt variants are using the tg commit but the jumpsuits are from vg..? It's fine if it is but it looks like something you missed.

Comment (Flareguy) [reply Resources/Textures/Clothing/Uniforms/Jumpskirt/color.rsi/meta.json:4]: The jumpsuits only use sprites from vgstation, but the jumpskirt uses specifically the skirt part from the tg commit. It's pretty awkward wording, but I'm not sure how else to phrase it - suggestions would be welcome.

Comment (UbaserB) [reply Resources/Textures/Clothing/Uniforms/Jumpskirt/color.rsi/meta.json:4]: "Taken from vgstation at commit https://github.com/vgstation-coders/vgstation13/commit/993de554ecfc30cc45f8340819c5a60ef1eee7e1 and modified by Flareguy & P3MOIRA, with the lower half of the jumpskirt taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039 and modified by Flareguy for SS14."

**Comment by UbaserB** [comment]
test fail real
